src/loaders/JsonImageInfoLoader.ts

```diff
@@ -276,6 +278,8 @@ class JsonImageInfoLoader extends ThreadableVolumeLoader {
         onData([chindex], [channelsBits[ch]], [DATARANGE_UINT8], [bitmap.width, bitmap.height]);
       }
     });
+
+    await Promise.all(imagePromises);
```

Comment:
The `Promise.all` has to stay to get the behavior this PR is trying to add, but the `await` is just there to consolidate the `undefined[]` produced by that promise into a single `undefined`. TS probably lets you assign `Promise<undefined[]>` to `Promise<void>` (the return type of this function) though, so that might let me just `return` this promise rather than `await` it. It's still gonna end up `await`ed somewhere up the call stack though.